app/src/main/res/layout/hero_item.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <ImageView
```

Comment:
android:scaleType="centerCrop"
En el caso de trabajar con imagenes en los items que pueden tener diferentes tamanños con esto obligamos a que ocupen todo el item y asi tener una vista mas regular de cara al usuario.

Reply:
That's right! I'm going to change it on the PR #14 :smile: